docs/features/mcp/README.md

```diff
@@ -0,0 +1,324 @@
+---
+post_title: Model Context Protocol (MCP) Integration
+sidebar_label: MCP Integration
+category_slug: mcp
+---
+
+# Model Context Protocol (MCP) Integration
+
+## Introduction
+
+WooCommerce includes native support for the Model Context Protocol (MCP), enabling AI assistants and tools to interact directly with WooCommerce stores through a standardized protocol. This integration exposes WooCommerce functionality as discoverable tools that AI clients can use to perform store operations with proper authentication and permissions.
+
+**Developer Preview Notice**: The MCP implementation in WooCommerce is currently in developer preview. Implementation details, APIs, and integration patterns may change in future releases as the feature matures.
```

Comment:
Feel free to wrap any of these notices in a note for a more visual callout on the docs site: 

```
:::info
...your notice here...
:::
```

More info here: https://docusaurus.io/docs/markdown-features/admonitions